profile_bluesky/startup/instrument/plans/xpcs_acquire_2.py

```diff
@@ -0,0 +1,68 @@
+
+
+
+"""
+Acquire an XPCS measurement with a supported area detector
+"""
+
+__all__ = """
+    AD_Acquire
```

Comment:
change to `Detector_Acq`

Reply:
otherwise, this file should NOT be changed before this PR is merged, even though the `preconfigure()` method is not complete

Reply:
This file is a *sketch* of code now, not actually used.  It is unrelated to the PR except for when we thought about it.  Got to write our thoughts somewhere.